runtime/hydradx/src/evm/runner.rs

```diff
@@ -0,0 +1,238 @@
+use crate::evm::WethAssetId;
```

Comment:
since most lines are copied from frontier, maybe we could acknowledge it in a comment?

Reply:
yeah, i would add it so we can upgrade it as well when frontier runner changes